rayepps/radash
# remove cdn version and cdn version check

## Description

We have two checks in CI to verify that the cdn files are up to date
- check that the version (`cdn/.version`) is the same as the package.json version
- run the build and hash the files and ensure they are identical

Obviously, one of these is useless and obsolete in compared to the other. I'm removing the cdn version, script, and check because it provides no value.

## Checklist

- [ ] Changes are covered by tests if behavior has been changed or added
- [ ] Tests have 100% coverage
- [ ] If code changes were made, the version in `package.json` has been bumped (matching semver)
- [ ] If code changes were made, the `yarn build` command has been run and to update the `cdn` directory

## Resolves

If the PR resolves an open issue tag it here. For example, `Resolves #34`
